<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationWindow</class>
 <widget class="QMainWindow" name="ApplicationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>text chewer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #AABDBA;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>-1</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="dish_path_btn">
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Courier New&quot;;</string>
       </property>
       <property name="text">
        <string>choose file
 path...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="dish_text">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: &quot;#E0E7E5&quot;; border: 2px solid rgb(119, 140, 125);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ready_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(129, 162, 166);
font: bold 14.5pt &quot;Courier New&quot;;</string>
    </property>
    <property name="text">
     <string>ready</string>
    </property>
   </widget>
   <widget class="QFrame" name="wordcloud_settings_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>461</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>461</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="wordcloud_settings">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="words_layout">
        <item>
         <widget class="QLabel" name="words_label">
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;Courier New&quot;; background-color: &quot;#B7C1BF&quot;;</string>
          </property>
          <property name="text">
           <string>words in
wordcloud</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="words_">
          <property name="styleSheet">
           <string notr="true">background-color: &quot;#CCD3D2&quot;;</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="height_layout">
        <item>
         <widget class="QLabel" name="height_label">
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;Courier New&quot;; background-color: &quot;#B7C1BF&quot;;</string>
          </property>
          <property name="text">
           <string>height of
wordcloud</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="height_">
          <property name="styleSheet">
           <string notr="true">background-color: &quot;#CCD3D2&quot;;</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="width_layout">
        <item>
         <widget class="QLabel" name="width_label">
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;Courier New&quot;; background-color: &quot;#B7C1BF&quot;;</string>
          </property>
          <property name="text">
           <string>width of
wordcloud</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="width_">
          <property name="styleSheet">
           <string notr="true">background-color: &quot;#CCD3D2&quot;;</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>1920</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>41</y>
       <width>459</width>
       <height>177</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="pofs_layout">
      <property name="horizontalSpacing">
       <number>-1</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="pofs_label">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Courier New&quot;; background-color: &quot;#B7C1BF&quot;;</string>
        </property>
        <property name="text">
         <string>parts of speech:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QVBoxLayout" name="pofs_checkboxes">
        <property name="spacing">
         <number>14</number>
        </property>
        <item>
         <widget class="QCheckBox" name="NOUN">
          <property name="text">
           <string>nouns</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ADJF">
          <property name="text">
           <string>full adjectives</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ADJS">
          <property name="text">
           <string>short adjectives</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="VERB">
          <property name="text">
           <string>personal verbs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="INFN">
          <property name="text">
           <string>infinitive verbs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="all_pofs_checkbox">
          <property name="text">
           <string>all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="setup_label_layout">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt bold &quot;Courier New&quot;;</string>
      </property>
      <property name="text">
       <string>set up your wordcloud:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="toggle_copying_checkbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Courier New&quot;;</string>
    </property>
    <property name="text">
     <string>save the resulting text to the clipboard</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
